src/internal/m365/collection/exchange/contacts_backup.go

```diff
@@ -29,6 +29,20 @@ func (h contactBackupHandler) itemHandler() itemGetterSerializer {
 	return h.ac
 }
 
+func (h contactBackupHandler) folderGetter() containerGetter {
+	return h.ac
+}
+
+func (h contactBackupHandler) previewIncludeFolders() []string {
```

Comment:
nit: We are using folders & containers interchangeably in a few places. e.g. `defaultPreviewContainerLimit`. Should we use container everywhere for consistency?